docs/usage/configuration-options.md

```diff
@@ -313,6 +313,14 @@ e.g. instead of `renovate/{{parentDir}}-`, configure the template part in `addit
     This setting does not change the default _onboarding_ branch name, i.e. `renovate/configure`.
     If you wish to change that too, you need to also configure the field `onboardingBranch` in your global bot config.
 
+## branchPrefixOld
+
+Renovate checks branch names to see if an update PR was created before.
+Open PR will continue to be updated until it is merged or closed unmerged.
+If you closed this PR, it is ignored and won't be recreated.
+Use this option to ignore PR created with `branchPrefixOld`.
+Best used with the `branchPrefix` setting.
```

Comment:
line 321 might need a change as well, seems off in this context
WDYT ?

Reply:
I'm not happy with the new texts. The sentences don't feel properly linked together conceptually. So the whole section needs to be looked at and fixed. 😉 

I'll let the maintainers check the technical accuracy first, before I make suggestions that don't match the code. 🙈

Reply:
i think this is technically right, but can be more detailed.